torch/nn/modules/module.py

```diff
@@ -1426,6 +1429,26 @@ def _register_load_state_dict_pre_hook(self, hook, with_module=False):
         self._load_state_dict_pre_hooks[handle.id] = hook
         return handle
 
+    def register_load_state_dict_post_hook(self, hook):
+        r"""
+        These hooks will be called with the following arguments:
```

Comment:
I would model this doc following the regular hooks rather than the private API doc above. The way they present the what it does / the signature / return seems clearer.
Also, if there is no benefit at passing the named tuple directly, it might be simpler to pass each list as an argument to make the signature clearer?

Example doc:
https://github.com/pytorch/pytorch/blob/7203a73986943c50b8d2f0fdc9d09821705a1602/torch/nn/modules/module.py#L1081-L1098
And the rendered version: https://pytorch.org/docs/stable/generated/torch.nn.Module.html#torch.nn.Module.register_forward_hook

Reply:
Agree with the doc change - I'll also suggest that a phrase or so be included that explicitly explains the relationship between these hooks and the `strict` flag, for clarity.